### ISSUE TITLE
Get rid of console warnings

### ISSUE THREAD

**Issue by humphd** [mozilla/brackets#700]
I'm sick of seeing these deprecation warnings.  We need to modify `extensions/default/brackets-paste-and-indent/main.js` to not do what it's complaining about, and/or get rid of the `console.warn()` completely.

```
Deprecated: Do not use $().on/off() on Brackets modules and model objects. Call on()/off() directly on the object without a $() wrapper.
    at new jQuery.fn.init (http://localhost:8000/dist/main.js:46:3999)
    at n (http://localhost:8000/dist/thirdparty/thirdparty.min.js:5:18091)
    at Object.<anonymous> (http://localhost:8000/dist/extensions/default/brackets-paste-and-indent/main.js:1:431)
    at Object.execCb (http://localhost:8000/dist/thirdparty/require.min.js:1:11627)
    at Z.check (http://localhost:8000/dist/thirdparty/require.min.js:1:6018)
    at Z.enable (http://localhost:8000/dist/thirdparty/require.min.js:1:8524)
    at Z.init (http://localhost:8000/dist/thirdparty/require.min.js:1:5181)
    at E (http://localhost:8000/dist/thirdparty/require.min.js:1:3775)
    at Object.completeLoad (http://localhost:8000/dist/thirdparty/require.min.js:1:10932)
i @ main.js:12
jQuery.fn.init @ main.js:46
n @ thirdparty.min.js:5
(anonymous) @ main.js:1
execCb @ require.min.js:1
check @ require.min.js:1
enable @ require.min.js:1
init @ require.min.js:1
E @ require.min.js:1
completeLoad @ require.min.js:1
onScriptLoad @ require.min.js:1
main.js:12 Registering for deprecated event 'currentDocumentChange'. Instead, use MainViewManager.currentFileChange. Error
    at Object.c [as on] (http://localhost:8000/dist/main.js:12:9086)
    at Object.<anonymous> (http://localhost:8000/dist/extensions/default/brackets-paste-and-indent/main.js:1:436)
    at Object.execCb (http://localhost:8000/dist/thirdparty/require.min.js:1:11627)
    at Z.check (http://localhost:8000/dist/thirdparty/require.min.js:1:6018)
    at Z.enable (http://localhost:8000/dist/thirdparty/require.min.js:1:8524)
    at Z.init (http://localhost:8000/dist/thirdparty/require.min.js:1:5181)
    at E (http://localhost:8000/dist/thirdparty/require.min.js:1:3775)
    at Object.completeLoad (http://localhost:8000/dist/thirdparty/require.min.js:1:10932)
    at HTMLScriptElement.onScriptLoad (http://localhost:8000/dist/thirdparty/require.min.js:1:11752)
```

**Comment by cgsingh** [comment]
I'll do this by just getting rid of the warning messages altogether.

**Comment by cgsingh** [comment]
I believe I have a solution ready, but I'm not entirely confident about my replacement for `currentDocumentChange`. I'm just going by what the documentation in `DocumentManager.js` suggests. I'll open a PR soon.

**Comment by cgsingh** [comment]
Uh, Git won't let me add changes I made to `brackets-paste-and-indent/main.js`, I guess because it's a module? Anyone know how to get around this?

**Comment by cgsingh** [comment]
I'm also getting some errors with Travis CI, so clearly I forgot to do something. Here is the PR: [](https://github.com/mozilla/brackets/pull/708)

**Comment by shikhar-scs** [comment]
Has this been solved completely in #781 ? If no, can I take over the progress made and approach suggested in #708?

**Comment by humphd** [comment]
@shikhar-scs correct, it's done by https://github.com/mozilla/brackets/pull/781.  Thanks for letting us know this is still open.
